controls/limit.sp

```diff
@@ -36,11 +36,174 @@ benchmark "limit" {
   title       = "Limit"
   description = "The number of tags on each resource should be monitored to avoid hitting the limit unexpectedly."
   children = [
+    control.api_management_tag_limit,
+    control.app_service_environment_tag_limit,
+    control.app_service_function_app_tag_limit,
+    control.app_service_plan_tag_limit,
+    control.app_service_web_app_tag_limit,
+    control.application_security_group_tag_limit,
+    control.batch_account_tag_limit,
+    control.compute_availability_set_tag_limit,
+    control.compute_disk_tag_limit,
+    control.compute_disk_encryption_set_tag_limit,
+    control.compute_image_tag_limit,
+    control.compute_snapshot_tag_limit,
     control.compute_virtual_machine_tag_limit,
-    control.storage_storage_account_tag_limit
+    control.compute_virtual_machine_scale_set_tag_limit,
+    control.container_registry_tag_limit,
+    control.cosmosdb_account_tag_limit,
+    control.cosmosdb_mongo_database_tag_limit,
+    control.cosmosdb_sql_database_tag_limit,
+    control.data_factory_tag_limit,
+    control.data_lake_analytics_account_tag_limit,
+    control.data_lake_store_tag_limit,
+    control.eventhub_namespace_tag_limit,
+    control.express_route_circuit_tag_limit,
+    control.firewall_tag_limit,
+    control.iothub_tag_limit,
+    control.key_vault_tag_limit,
+    control.key_vault_deleted_vault_tag_limit,
+    control.key_vault_key_tag_limit,
+    control.key_vault_managed_hardware_security_module_tag_limit,
+    control.key_vault_secret_tag_limit,
+    control.kubernetes_cluster_tag_limit,
+    control.lb_tag_limit,
+    control.log_alert_tag_limit,
+    control.log_profile_tag_limit,
+    control.logic_app_workflow_tag_limit,
+    control.mariadb_server_tag_limit,
+    control.mssql_elasticpool_tag_limit,
+    control.mssql_managed_instance_tag_limit,
+    control.mysql_server_tag_limit,
+    control.network_interface_tag_limit,
+    control.network_security_group_tag_limit,
+    control.network_watcher_tag_limit,
+    control.network_watcher_flow_log_tag_limit,
+    control.postgresql_server_tag_limit,
+    control.public_ip_tag_limit,
+    control.recovery_services_vault_tag_limit,
+    control.redis_cache_tag_limit,
+    control.resource_group_tag_limit,
+    control.route_table_tag_limit,
+    control.search_service_tag_limit,
+    control.servicebus_namespace_tag_limit,
+    control.sql_database_tag_limit,
+    control.sql_server_tag_limit,
+    control.storage_account_tag_limit,
+    control.stream_analytics_job_tag_limit,
+    control.virtual_network_tag_limit,
+    control.virtual_network_gateway_tag_limit
   ]
 }
 
+control "api_management_tag_limit" {
+  title       = "API managements should not exceed tag limit"
```

Comment:
Is there a resource we can use in the title and description, like API Management services?